sql-database/create-and-configure-database/create-and-configure-database.sh

```diff
@@ -1,47 +1,56 @@
 #!/bin/bash
 
+# set execution context (if necessary)
+az account set --subscription <replace with your subscription name or id>
+
+# Set the resource group name and location for your server
+export resourceGroupName=myResourceGroup-$RANDOM
+export location=westus2
```

Comment:
Default location looks like it's currently overloaded for SQL DBs:

```
Deployment failed. Correlation ID: 3d59ee50-e3d9-4c07-97cf-e1856dba8675.
Location 'West US 2' is not accepting creation of new Windows Azure SQL
Database servers at this time.
```

Reply:
succeeds for me 5 minutes ago

Reply:
and again succeeds now 3:14 PM PDT